### PR TITLE
Generalize description of MMU MCU power errors

### PR DESCRIPTION
**Pull request by DRracer**
as requested in Prusa-Error-Codes PR#97

There will be more separate sources of MCU power errors in the future and reporting each of them separately doesn't make much sense - especially when the only thing a user can do about it is to check the connectors.

So based on this, the error title has changed a bit (we are not using the full text description in 8bit FW)

MMU-254

### CODE REVIEW

File: lang/po/Firmware_it.po [hunk 1]
```diff
@@ -2490,10 +2490,10 @@ msgstr ""
 "Caricamento nell' estrusore non riuscito. Ispezionare la forma della punta "
 "del fil. Affinare la calib. del sensore, se necessario."
 
-#. MSG_TITLE_MCU_UNDERVOLTAGE_VCC c=20
+#. MSG_TITLE_MCU_ERROR c=20
 #: ../../Firmware/mmu2/errors_list.h:176 ../../Firmware/mmu2/errors_list.h:222
-msgid "MMU MCU UNDERPOWER"
-msgstr "SOTTOTENSION MCU VCC"
+msgid "MMU MCU ERROR"
+msgstr "MMU MCU ERRORE"
```
Comment (wavexx) [inline lang/po/Firmware_it.po]: Can't say for the others, but this is better written as "ERRORE MMU MCU"

Comment (DRracer) [reply lang/po/Firmware_it.po]: fixed [3b9dc97](https://github.com/prusa3d/Prusa-Firmware/pull/4269/commits/3b9dc97ec673ebb4fbb30c0a67286a4dfd67b675)

File: lang/po/Firmware_ro.po [hunk 2]
```diff
@@ -2489,10 +2489,10 @@ msgstr ""
 "Încărcarea în extruder nu a reușit. Inspectați forma vârfului filamentului. "
 "Rafinați calibrarea senzorului, dacă este necesar."
 
-#. MSG_TITLE_MCU_UNDERVOLTAGE_VCC c=20
+#. MSG_TITLE_MCU_ERROR c=20
 #: ../../Firmware/mmu2/errors_list.h:176 ../../Firmware/mmu2/errors_list.h:222
-msgid "MMU MCU UNDERPOWER"
-msgstr "MCU SUBTENSIUNE VCC"
+msgid "MMU MCU ERROR"
+msgstr "MMU MCU EROARE"
```
Comment (leptun) [inline lang/po/Firmware_ro.po]: I think `MMU EROARE MCU` might sound a bit better.

Comment (DRracer) [reply lang/po/Firmware_ro.po]: fixed [ca478ea](https://github.com/prusa3d/Prusa-Firmware/pull/4269/commits/ca478ea07c2d911b82c67f21568347255385cb14)

File: Firmware/mmu2/errors_list.h [hunk 3]
```diff
@@ -240,7 +240,7 @@ static const char MSG_DESC_FSENSOR_DIDNT_TRIGGER[] PROGMEM_I1 = ISTR("Filament s
 static const char MSG_DESC_FSENSOR_FILAMENT_STUCK[] PROGMEM_I1 = ISTR("Filament sensor didn't switch off while unloading filament. Ensure filament can move and the sensor works."); ////MSG_DESC_FSENSOR_FILAMENT_STUCK c=20 r=8
 static const char MSG_DESC_PULLEY_CANNOT_MOVE[] PROGMEM_I1 = ISTR("Pulley motor stalled. Ensure the pulley can move and check the wiring."); ////MSG_DESC_PULLEY_CANNOT_MOVE c=20 r=8
 static const char MSG_DESC_FSENSOR_TOO_EARLY[] PROGMEM_I1 = ISTR("Filament sensor triggered too early while loading to extruder. Check there isn't anything stuck in PTFE tube. Check that sensor reads properly."); ////MSG_DESC_FSENSOR_TOO_EARLY c=20 r=8
-static const char MSG_DESC_INSPECT_FINDA[] PROGMEM_I1 = ISTR("Selector can't move due to FINDA detecting a filament. Make sure no filament is in selector and FINDA works properly."); ////MSG_DESC_INSPECT_FINDA c=20 r=8
+static const char MSG_DESC_INSPECT_FINDA[] PROGMEM_I1 = ISTR("Selector can't move due to FINDA detecting a filament. Make sure no filament is in Selector and FINDA works properly."); ////MSG_DESC_INSPECT_FINDA c=20 r=8
```
Comment (wavexx) [inline Firmware/mmu2/errors_list.h:243]: Do we want to update the POT file yet for this?

Comment (DRracer) [reply Firmware/mmu2/errors_list.h:243]: fixed [46197de](https://github.com/prusa3d/Prusa-Firmware/pull/4269/commits/46197de9af349d36910e44ae1e1ba712331458f1)

**Review by wavexx — APPROVED** [review]
Builds clean now

**Comment by DRracer** [comment]
squashed the minor fixup commits into [f05d3e1](https://github.com/prusa3d/Prusa-Firmware/pull/4269/commits/f05d3e19a5b2f2fa442cd30493990c49d9e7c143)

**Review by leptun — APPROVED** [review]

**Review by gudnimg — APPROVED** [review]
Looks good to me 👍

**Pull request merged**